### ISSUE TITLE
cellStylesSticky zIndex doesn't work properly with Material UI Table with stickyHeader prop

### ISSUE THREAD

**Issue by sherbakov1** [GuillaumeJasmin/react-table-sticky#4]
The current zIndex interferes with MaterialUI Table with stickyHeader prop.
```
const cellStylesSticky = {
  position: 'sticky',
  zIndex: 2,
};
```

Please, increase the zIndex to 3 to solve the issue.

<img width="121" alt="Screenshot 2020-03-17 at 11 41 41" src="https://user-images.githubusercontent.com/16952985/76844230-ab0cdb00-6845-11ea-9e3c-f748d9ecbb84.png">
<img width="243" alt="Screenshot 2020-03-17 at 11 41 53" src="https://user-images.githubusercontent.com/16952985/76844234-ac3e0800-6845-11ea-955c-cea0be16d197.png">

**Comment by GuillaumeJasmin** [comment]
@sherbakov1 thanks for your feedback :) 
Please provide more informations (or better, a codesandbox example).
What is the current behavior ? Does Material UI override zIndex ?

**Comment by sherbakov1** [comment]
@GuillaumeJasmin Thanks for the quick reply! Please, check the images above :)

**Comment by GuillaumeJasmin** [comment]
Ok I get the issue !
I have release `1.1.0` yesterday with new css selector: https://github.com/GuillaumeJasmin/react-table-sticky/releases/tag/v1.1.0

You can temporary try:
```
[data-sticky-td] {
    position: sticky;
    zIndex: 3 !important;
}
```

I will make a new release today with full CSS support and I think remove the inline zIndex

**Comment by sherbakov1** [comment]
Awesome! Great job! I will wait for the release.

**Comment by GuillaumeJasmin** [comment]
fixed on `v1.1.2` with `z-index: 3` . Sorry for the late publish.
In the next major version, I will remove hard coded inline style and use only CSS.
Thanks for your help :)
